conformance/operator_test.go

```diff
@@ -41,10 +41,16 @@ var _ = Describe("operator", func() {
 		Expect(err).ToNot(HaveOccurred())
 
 		Eventually(func() bool {
+			isClusterReady, err := cluster.IsClusterStable(clients)
```

Comment:
Can we have this and-ed with sriov stable?
Here, if clusterIsReady we return even though sriovstable could be false.